nestauk/afs_early_years_labour_market_analysis
# [1] Finding relevant jobs in OJO

---

# Description

A first pass at trying to find job adverts in the OJO data. Looks at a couple of things;

- Whether the "sector" is Nursery
- Whether the job title matches a list of predefined job titles
- Whether the occupation matches one of the most frequently mentioned occupations in the nursery sector (i.e. cases where the sector hasn't been defined but occupation has)
- Whether "early years" is in the job title

Also adds getters for:
1. All job adverts
2. Relevant job adverts

Closes #1 

# Instructions for Reviewer

In order to test the code in this PR you need to:

1. Install the requirements.txt
2. From the project root, run `python afs_early_years_labour_market_analysis/pipeline/data_collection/refine_relevant_jobs.py run`, which runs a metaflow flow, looking for relevant job adverts
3. Check the getters work by importing them into a notebook and checking they run

# Checklist:
